devfile.yaml

```diff
@@ -38,7 +38,7 @@ commands:
                                                                elif [[ -d /projects/target/liberty/wlp && ! -e /projects/.liberty-mv ]];
                                                                  then echo "STACK WARNING - LIBERTY RUNTIME WAS LOADED FROM HOST";
                                                                fi
-                                                            && mvn -Dliberty.runtime.version=20.0.0.10 package
+                                                            && mvn -Dliberty.runtime.version=20.0.0.10 -Dmaven.compiler.target=11 -Dmaven.compiler.source=11 package
```

Comment:
While it's nice that we supply a default for projects that don't have one (see https://github.com/OpenLiberty/application-stack/wiki/Common-Problems) I don't think it's ideal that this would override what's in the user's pom.xml.